Documentation/Contributing/UnitTests.md

```diff
@@ -77,55 +76,107 @@ To create a new test the following template can be used:
+        // This method is called once before we enter play mode and executes any of the tests
+        // Do any kind of setup here that can't be done in playmode
+        [SetUp]
+        public virtual void Setup()
         {
-            // eg installing unity packages is only possible in edit mode
-            // so if a test requires TextMeshPro we will need to check for the package before entering play mode
-            PlayModeTestUtilities.EnsureTextMeshProEssentials();
+            // NOTE: This setup initializes the main camera at Vector3(1.0f, 1.5f, -2.0f)
+            PlayModeTestUtilities.Setup();
         }
 
-        // do common setup for each of your tests here - this will be called for each individual test after entering playmode
-        [SetUp]
-        public void Init()
+        // Destroy commonly initialized objects here - this will be called after each of your tests has finished
+        [TearDown]
+        public virtual void TearDown()
         {
-            // in most play mode test cases you would want to at least create an MRTK GameObject using the default profile
-            TestUtilities.InitializeMixedRealityToolkit(true);
+            PlayModeTestUtilities.TearDown();
         }
+    }
+}
+#endif
 
+```
 
-        // destroy commonly initialized objects here - this will be called after each of your tests has finished
-        [TearDown]
-        public void Shutdown()
+## Creating a new play mode test
+To create a new play mode test:
+- Navigate to Assets > MixedRealityToolkit.Tests > PlayModeTests
+- Right click, Create > Testing > C# Test Script
+- Replace the default template with the example starter test below
+
+``` csharp
+#if !WINDOWS_UWP
+// When the .NET scripting backend is enabled and C# projects are built
+// The assembly that this file is part of is still built for the player,
+// even though the assembly itself is marked as a test assembly (this is not
+// expected because test assemblies should not be included in player builds).
+// Because the .NET backend is deprecated in 2018 and removed in 2019 and this
+// issue will likely persist for 2018, this issue is worked around by wrapping all
+// play mode tests in this check.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using NUnit.Framework;
+using System;
+using System.Collections;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Microsoft.MixedReality.Toolkit.Tests
+{
+    class ExamplePlayModeTests : BasePlayModeTests
+    {
+        // Override Setup() in BasePlayModeTests to change the starting camera position
+        public override void Setup()
         {
-            // call shutdown if you've created an mrtk GameObject in your test
-            TestUtilities.ShutdownMixedRealityToolkit();
+            base.Setup();
+            
+            // Set camera position to Vector3(0,0,0), the default position Vector3(1.0f, 1.5f. -2.0f)
```

Comment:
Within `PlayModeTestUtilities.Setup()` the camera position is set to (1, 1.5, -2). The purpose of the `TestUtilities.PlayspaceToOriginLookingForward()` is to set the position of the camera for the tests to (0, 0, 0), which is where I initially assumed it was placed.  That comment should be more like: 

`\\ Change the position of the main camera to (0, 0, 0), in PlayModeTestUtilites.Setup() the camera is set to position (1, 1.5, -2)` 

But it may not be necessary to add it, the reason for including it was because I assumed the camera was already at the origin and ran into issues because it was not.